src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -72,37 +72,38 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
     >
       <div className={classNames('govuk-grid-row', styles.releaseIntro)}>
         <div className="govuk-grid-column-two-thirds">
-          <div className="dfe-flex dfe-align-items--center dfe-justify-content--space-between">
-            <div className="dfe-flex govuk-!-margin-bottom-3">
+          <div className="dfe-flex dfe-align-items--center dfe-justify-content--space-between govuk-!-margin-bottom-3">
+            <div>
               {release.latestRelease ? (
-                <Tag strong className="govuk-!-margin-right-6">
+                <Tag className="govuk-!-margin-right-3 govuk-!-margin-bottom-3">
                   This is the latest data
                 </Tag>
               ) : (
-                <Link
-                  className="dfe-print-hidden"
-                  unvisited
-                  to="/find-statistics/[publication]"
-                  as={`/find-statistics/${release.publication.slug}`}
-                >
-                  View latest data:{' '}
-                  <span className="govuk-!-font-weight-bold">
-                    {release.publication.otherReleases[0].title}
-                  </span>
-                </Link>
+                <div className="govuk-!-margin-bottom-3">
+                  <Link
+                    className="dfe-print-hidden "
```

Comment:
Looks like you might have tried this already, but can't `govuk-!-margin-bottom-3` go in here instead of on a wrapping div?